src/Services/ValidationService.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace ITBugTracking\Services;
+
+use ITBugTracking\Hydrators\SeverityHydrator;
+use PDO;
+
+class ValidationService
+{
+
+    public static function checkRequiredDataExists($data): bool
+    {
+        return !empty($data['reporter']) || !empty($data['title']) || !empty($data['severity']) || !empty($data['department']);
+    }
+
+
+    //sanitize 255 character limits
+    public static function limitTitleCharLengthTo255($data)
+    {
+        if (isset($data)) {
+
+            if (strlen($data) > 255) {
+                $data = substr($data, 0, 255);
+            }
+            return $data;
+        }
+    }
+
+    public static function limitReporterCharLengthTo255($data)
+    {
+        if (strlen($data) > 255) {
+            $data = substr($data, 0, 255);
+        }
+        return $data;
+
+    }
+
+    // 100000 character limit for description
+    public static function descriptionLimitCharLength($data)
```

Comment:
```suggestion
    public static function limitDescriptionLength(string $data)
```

Similar functions should have similar names.

If you want a re-usable function you could use:

```php
public static function limitStringLength(string $string, int $limit) {
    $output = trim($string);
    return substr($output, 0, $limit);
}
```

Reply:
done